inc/zoo/swar/math.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+#include "SWAR.h"
+#include <cstddef>
+#include <cstdint>
+
+namespace zoo::math {
```

Comment:
You know, should you have used `maths`, I would have let it slide.  But since you didn't, if you try to use the British `maths` I will reject the renaming on the basis that there is no need to make a change like that
;-)

Reply:
I wouldn't have let 'maths' slide.